lockfile/verification/test/satisfiesPackageManifest.ts

```diff
@@ -77,7 +77,9 @@ test('satisfiesPackageManifest()', () => {
     }
   )).toStrictEqual({
     satisfies: false,
-    detailedReason: 'specifiers in the lockfile ({"foo":"^1.0.0"}) don\'t match specs in package.json ({"foo":"^1.1.0"})',
+    detailedReason: `specifiers in the lockfile don't match specifiers in package.json:
+* 1 dependencies were changed: foo (^1.0.0 → ^1.1.0)
```

Comment:
Maybe not that it was changed in package.json
```suggestion
    detailedReason: `specifiers in the lockfile don't match specifiers in package.json:
* 1 dependencies were changed in `package.json`: foo (^1.0.0 → ^1.1.0)
```